### PR TITLE
Fix #15

### LINKED ISSUE #15: Create project base
**Is your feature request related to a problem? Please describe.**
We need a base for start coding into.

**Describe the solution you'd like**
A boilerplate for start coding into, including npm configs (package.json) and others like babel or folders (src, public, etc).

**Describe alternatives you've considered**
Create a minimal node babel server.

**Additional context**
https://www.robinwieruch.de/minimal-node-js-babel-setup/

### PR DESCRIPTION
**Pull request by jadsonluan**
Create a minimal node babel server

### CODE REVIEW

**Comment by LukeHxH** [comment]
Vou avaliar já já, só clonar aqui rapidinho pra testar.

**Comment by LukeHxH** [comment]
When I ran, it gave me a compilation error because of dotenv. Do I need to create an environment file or something to run?

Error:
```
/home/lukehxh/Git/server/src/index.js:1
(function (exports, require, module, __filename, __dirname) { import 'dotenv/config';
                                                                     ^^^^^^^^^^^^^^^

SyntaxError: Unexpected string
    at new Script (vm.js:85:7)
    at createScript (vm.js:266:10)
    at Object.runInThisContext (vm.js:314:10)
    at Module._compile (internal/modules/cjs/loader.js:698:28)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:749:10)
    at Module.load (internal/modules/cjs/loader.js:630:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:570:12)
    at Function.Module._load (internal/modules/cjs/loader.js:562:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:801:12)
    at internal/main/run_main_module.js:21:11
```

**Comment by LukeHxH** [comment]
Besides that, I think that we should use:

```js
import { VariableName } from 'dotenv/config';
```

instead of:

```js
import 'dotenv/config';
```

File: package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "main": "index.js",
+  "scripts": {
+    "start": "nodemon --exec babel-node src/index.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/panelinhadees/server.git"
+  },
+  "author": "Jadson Luan <jadsonluan1@gmail.com>",
```
Comment (fanny) [inline package.json]:
```suggestion
  "author": "Jadson Luan <jadsonluan1@gmail.com>;Fanny Vieira<fannyvieira082@gmail.com>;Lucas Medeiros<lucas.medeiros@ccc.ufcg.edu.br>"
```

**Comment by fanny** [comment]
> When I ran `index.js`, it gave me a compilation error because of dotenv. Do I need to create an environment file or something to run?
> 
> Error:
> 
> ```
> /home/lukehxh/Git/server/src/index.js:1
> (function (exports, require, module, __filename, __dirname) { import 'dotenv/config';
>                                                                      ^^^^^^^^^^^^^^^
> 
> SyntaxError: Unexpected string
>     at new Script (vm.js:85:7)
>     at createScript (vm.js:266:10)
>     at Object.runInThisContext (vm.js:314:10)
>     at Module._compile (internal/modules/cjs/loader.js:698:28)
>     at Object.Module._extensions..js (internal/modules/cjs/loader.js:749:10)
>     at Module.load (internal/modules/cjs/loader.js:630:32)
>     at tryModuleLoad (internal/modules/cjs/loader.js:570:12)
>     at Function.Module._load (internal/modules/cjs/loader.js:562:3)
>     at Function.Module.runMain (internal/modules/cjs/loader.js:801:12)
>     at internal/main/run_main_module.js:21:11
> ```
> 
> Besides that, I think that we should use:
> 
> ```js
> import { VariableName } from 'dotenv/config';
> ```
> 
> instead of:
> 
> ```js
> import 'dotenv/config';
> ```

Do you run `npm install`?

**Comment by LukeHxH** [comment]
@fanny Yes, I did.

**Comment by jadsonluan** [comment]
I compiled again and no problem appeared. @fanny can you try?

**Comment by jadsonluan** [comment]
@LukeHxH , you should run using **npm start**. This way the babel can run and compile the code.

**Comment by fanny** [comment]
Exact, @jadsonluan. You create a `npm start` script to inialize the app, then run `node index.js` does not work, @LukeHxH.

For me, this pr is valid, making corrections on author list of `package.json`

**Comment by LukeHxH** [comment]
@jadsonluan Understood!

**Pull request merged**